Assets/Scripts/UI/Camera/SpectatorMovement.cs

```diff
@@ -6,13 +6,21 @@ public class SpectatorMovement : MonoBehaviour
     public bool disable;
     private float speed = 100f;
 
+    private void reset()
```

Comment:
Name should be `reset`. Please see: https://www.dofactory.com/reference/csharp-coding-standards